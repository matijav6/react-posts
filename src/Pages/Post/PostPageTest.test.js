import renderer from 'react-test-renderer';
import {PostPage} from './';

test('Post page renders correctly', () => {
    let location = {state: {post: {comments: [], user: {}, body: '', title: '', id: 0, userId: 0}}};
    let wrapper = renderer.create(<PostPage hello={''} location={location} sendRequest={() => {}}/>);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
});
