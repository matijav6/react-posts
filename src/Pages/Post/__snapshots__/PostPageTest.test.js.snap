// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post page renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-dark bg-primary d-flex justify-content-center min-height-50"
  />,
  <div
    className="container-fluid p-3"
  >
    <div
      className="row justify-content-md-center"
    >
      <div
        className="col-sm-8 mb-4 d-flex justify-content-center"
      >
        <div
          className="w-100"
        >
          <div
            className="sc-bdfBwQ fUVpNk card h-auto mw-100"
          >
            <div
              className="sc-gsTCUz fjdwEL card__body"
            >
              <h1
                className="sc-dlfnbm kGyIFc card__title"
              >
                
              </h1>
              <p
                className="sc-hKgILt domCRS card__text"
              >
                
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="back-button alert-primary"
    onClick={[Function]}
  >
    <h5>
      &lt;-
    </h5>
  </div>,
]
`;
